packages/react-devtools-shared/src/__tests__/inspectedElement-test.js

```diff
@@ -459,7 +459,7 @@ describe('InspectedElement', () => {
     const prevInspectedElement = inspectedElement;
 
     // This test causes an intermediate error to be logged but we can ignore it.
-    console.error = () => {};
```

Comment:
This caused the error prior to https://github.com/facebook/react/pull/24084 to be hidden